### PR TITLE
A first pull request on Capstone Project II

### PR DESCRIPTION
**Pull request by dlwub**
In this project:
- JavaScript has been used to build single pages dynamically 
- ES6 syntax and ES6 modules have been used
- Callbacks and promises have been in use
- webpack is used 
- Unit tests have been developed for testing movieCount and commentCount functions
- Gitflow used
- Data has been sent to and received from the TVmaze and Involvement APIs
- JavaScript code is made asynchronous.

### CODE REVIEW

File: src/modules/postLikes.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  if (e.target.classList.contains('likes-btn')) {
+    const url = `${getUrl()}/likes`;
+    const data = {
+      item_id: `movie${e.target.id.replace('like-btn-', '')}`,
+    };
+    toAPI(url, data);
+    e.target.children[0].innerHTML = '<i class="fa-solid fa-heart" style="color:red"></i>';
+    e.target.children[1].innerHTML = parseInt(e.target.children[1].innerHTML, 10) + 1;
+  }
+};
```
Comment (EricMbouwe) [inline src/modules/postLikes.js:14]: - [ ] When a user clicks on the Like button of an item, the interaction should get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), and the screen gets updated. When a user clicks on the Like button of an item, the interaction should get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270), and the screen gets updated.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @dlwub, 

Good job so far! 🔥
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes ♻♻
- [ ] Let's add a video presenting the project's features.
- [ ] Let's add A popup window with more data about an item that users can use to comment on. Currently, the comment section is shown below the list
- [ ] Your app should contain a `Home page header` and navigation similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)It contains a Home page header and navigation similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md)

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fix your home page to eliminate the full-screen white space at the beginning ♻

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/displayPopup.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      toAPI(url, newComment);
+      updateComment(id, name, comment);
+    }
+
+    document.getElementById(`name-input${id}`).value = '';
+    document.getElementById(`comment-input${id}`).value = '';
+  });
+};
+
+export default displayPopup;
```
Comment (ahmetbozaci) [inline src/modules/displayPopup.js:51]: _I make it optional but it is a requirement that not mention in the last review_
- [ ] I can't see pictures of the movie in the comment popup. Kindly fix it.
- [ ] Also ı can't see the close button or mark on the popup. Kindly add it
image

File: src/modules/postLikes.js (hunk 1 above)
Comment (ahmetbozaci) [reply src/modules/postLikes.js:14]: - [ ] Like button is not working always. Kindly fix it.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 


@dlwub @X-Elie-X  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
**- Like the last code reviewer said;**
- [ ] Kindly delete the white space on the page. It has also in the comment popup. I have to scroll down to see the movies.
image
image
- [ ] You add a presentation video link but google drive wants to access it. Kindly open access so we can watch it. 
_Next code reviewer kindly check the video if the video was made by following the requirements_
- [ ] Your app should contain a navigation bar like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/images/Home.png). Kindly add it. 
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by dlwub** [comment]
The link for the live demo works perfectly from our end but the reviewers had a problem opening it properly. Can you please open it in incognito mode?

**Comment by ahmetbozaci** [comment]
_The next code reviewer. Kindly approve it._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi team😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done making the changes requested from the previous reviewer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**